packages/airnode-deployer/src/cli/index.ts

```diff
@@ -183,6 +182,40 @@ yargs(hideBin(process.argv))
       }
     }
   )
+  .command(
+    'fetch-files <deployment-id> [version-id]',
+    'Fetch deployment files for the deployed Airnode',
+    (yargs) => {
+      yargs.positional('deployment-id', {
+        description: `ID of the deployment (from 'list' command)`,
+        type: 'string',
+      });
+      yargs.positional('version-id', {
+        description: `ID of the deployment version (from 'info' command)`,
+        type: 'string',
+      });
+      yargs.option('output-dir', {
+        alias: 'o',
+        description: 'Where to store fetched files',
+        default: 'config/',
```

Comment:
I prefer explicit setting of `output` locations when commands generate/download stuff, but we do provide default for `deploy` command.